src/main/java/thePackmaster/cards/bladestormpack/TradeWinds.java

```diff
@@ -0,0 +1,61 @@
+package thePackmaster.cards.bladestormpack;
+
+import com.evacipated.cardcrawl.mod.stslib.patches.FlavorText;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.CardGroup;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import thePackmaster.actions.FlexibleDiscoveryAction;
+import thePackmaster.actions.bladestormpack.TradeWindsAction;
+import thePackmaster.util.creativitypack.JediUtil;
+
+import java.util.ArrayList;
+import java.util.stream.Collectors;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+import static thePackmaster.cards.bladestormpack.FlavorConstants.*;
+
+/*REFS: AttackVial (womaninbluepack), StrikingStrike (strikepack), Mimicry & Paintbrush (creativitypack),
+RazorWind (dragonwrathpack), DualHeal (summonerspellspack), Maddii (Discord) */
+public class TradeWinds extends AbstractBladeStormCard {
+    public final static String ID = makeID("TradeWinds");
+    private static final int COST = 0;
+    private static final int HEAL_IF_COMMON = 2;
+    private static final int GOLD_LOSS_IF_RARE = 5;
+
+    public TradeWinds() {
+        super(ID, COST, CardType.SKILL, CardRarity.UNCOMMON, CardTarget.SELF);
+        baseMagicNumber = magicNumber = HEAL_IF_COMMON;
+        baseSecondMagic = secondMagic = GOLD_LOSS_IF_RARE;
+        exhaust = true;
+
```

Comment:
Since this card can provide healing, it should be tagged as healing so it can't be generated by in-combat card creation effects: `tags.add(CardTags.HEALING);`